src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -0,0 +1,585 @@
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_xxxxx CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_xxxxx CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_xxxxx CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_xxxxx CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_xxxxx CASCADE
+-- show that TRUNCATE is successfull
+SELECT COUNT(*) FROM ref_table, dist_table;
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+---------------------------------------------------------------------
+------ local execution of DROP ------
+---------------------------------------------------------------------
+-- droping just the referenced table would error out as dist_table references it
+DROP TABLE ref_table;
+ERROR:  cannot drop table ref_table because other objects depend on it
+-- drop those two tables via remote execution
+DROP TABLE ref_table, dist_table;
+-- drop the other standalone table locally
+DROP TABLE ref_table_1;
+NOTICE:  executing the command locally: DROP TABLE IF EXISTS local_commands_test_schema.ref_table_1_xxxxx CASCADE
+-- show that DROP commands are successfull
+SELECT tablename FROM pg_tables where schemaname='local_commands_test_schema' ORDER BY tablename;
+ tablename
+---------------------------------------------------------------------
+(0 rows)
+
+CREATE TABLE ref_table (a int primary key);
+SELECT create_reference_table('ref_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- We execute SELECT command within the below block locally.
+-- Hence we should execute the DROP command locally as well.
+BEGIN;
+  SELECT COUNT(*) FROM ref_table;
+NOTICE:  executing the command locally: SELECT count(*) AS count FROM local_commands_test_schema.ref_table_1500034 ref_table
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+  DROP TABLE ref_table;
+NOTICE:  executing the command locally: DROP TABLE IF EXISTS local_commands_test_schema.ref_table_xxxxx CASCADE
+COMMIT;
+CREATE TABLE ref_table (a int primary key);
+SELECT create_reference_table('ref_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE dist_table(a int);
+SELECT create_distributed_table('dist_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER TABLE dist_table ADD CONSTRAINT fkey FOREIGN KEY(a) REFERENCES ref_table(a);
+-- As both commands will prefer remote exeuction, TRUNCATE would be aware of
+-- dist_table is dropped. Hence, below block should not be effective as TRUNCATE
+-- will error out.
+BEGIN;
```

Comment:
I think we don't need to have tests that directly testing postgres such as this one

Reply:
Actually I wanted to show that they are aware of each other since they both preferred remote execution. Does this approach sound good ?

Reply:
But the error is coming from Postgres, before Citus kicks in. So, it doesn't add any value for us and could be confusing. I'd prefer to remove it